src/main/java/nextstep/courses/domain/session/sessionCoverImage/SessionCoverImage.java

```diff
@@ -0,0 +1,16 @@
+package nextstep.courses.domain.session.sessionCoverImage;
+
+public class SessionCoverImage {
+    private final String imageFileType;
```

Comment:
String 타입 대신 ImageType 을 활용하면 어떨까요?
```suggestion
    private final ImageType imageFileType;
```